docs/discover/set-time-filter.asciidoc

```diff
@@ -9,18 +9,19 @@ range in the histogram.
 
 
 [float]
-=== Filter with the time picker
```

Comment:
Would you mind making a few more changes to this doc?  Some suggestions:

1) Use "Setting the time range" as the title, or something similar
2) Reword the intro.  It still uses time picker.  Something like (not sure time range is correct)
You can set a time range if your index contains time-based events and a time-field is configured for the selected index pattern. By default, the time range is set to the last 15 minutes. You can use the time filter to change the time range, or select a specific time range in the histogram.

3) Rename "Specify the time range" to "Use the time filter"

4) Rewrite this sentence to flow with the rest of the text:  With the time filter, you can choose from

5) Rename histogram title to "Select a time range from the histogram"